### PR TITLE
Story1 fix edit applicant stage dropdown

### PR DESCRIPTION
**Pull request by Albertoroldanq**
Hi Rayna, as you said earlier, we added a fourth parameter to the stagesDropdown method and changed the logic that checked if the option matches with the current option in order to add the selected attribute in one of the `<option>` tags.

### CODE REVIEW

File: tests/ViewHelpers/ApplicantFormViewHelperTest.php [hunk 1]
```diff
@@ -15,17 +15,19 @@ public function testSuccessApplicantForm()
         ];
 
         $stageOptions = [
-            ['id' => 1, 'option' => 'dummy-option', 'stageId' => 1]
+            ['id' => 1, 'option' => 'dummy-option', 'stageId' => "1"],
```
Comment (norayna) [inline tests/ViewHelpers/ApplicantFormViewHelperTest.php:18]: why the quotes here?

File: src/ViewHelpers/ApplicantFormViewHelper.php [hunk 2]
```diff
@@ -4,11 +4,9 @@
 
 class ApplicantFormViewHelper
 {
-    public static function stagesDropdown($stages, $stageOptions, $currentStage)
+    public static function stagesDropdown($stages, $stageOptions, $currentStage, $currentOption)
     {
         $string = '';
-
-        /* Tech debt - to be tidied up */
```
[the remark this answers is not in the document]
Comment (Albertoroldanq) [reply src/ViewHelpers/ApplicantFormViewHelper.php:11]: we thought the tech debt was solving the issue

File: tests/ViewHelpers/ApplicantFormViewHelperTest.php (hunk 1 above)
Comment (Albertoroldanq) [reply tests/ViewHelpers/ApplicantFormViewHelperTest.php:18]: we added quotes to follow the same format as the student in stages array

**Review by norayna — CHANGES_REQUESTED** [review]
doesn't seem like the controller has been added to this pull request? the EditApplicantController is the same in master and in the story 1 branch. i thought you guys made a change to it?

**Comment by Albertoroldanq** [comment]
> doesn't seem like the controller has been added to this pull request? the EditApplicantController is the same in master and in the story 1 branch. i thought you guys made a change to it?

**Comment by Albertoroldanq** [comment]
> > doesn't seem like the controller has been added to this pull request? the EditApplicantController is the same in master and in the story 1 branch. i thought you guys made a change to it?

No, we didn't do anything in this fill, we just used the code that was already there to get the $data[currentOption]

File: src/ViewHelpers/ApplicantFormViewHelper.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (norayna) [reply src/ViewHelpers/ApplicantFormViewHelper.php]: ah fair lol, i think what they meant was that the code was kinda bad even without the bug 😅

**Review by norayna — APPROVED** [review]

**Pull request merged**